lib/hydra/derivatives/processors/jpeg2k_image.rb

```diff
@@ -73,20 +73,18 @@ def layer_rates(layer_count, compression_numerator)
         rates.map(&:to_s).join(',')
       end
 
-      protected
```

Comment:
The failing spec that he fixed had this error:

```
8) Transcoding with an attached tiff transcodes
     Failure/Error: long_dim = self.class.long_dim(image)
     
     NoMethodError:
       protected method `long_dim' called for Hydra::Derivatives::Processors::Jpeg2kImage:Class
```

"protected" for class methods apparently doesn't behave in the way you might expect.